src/harness/testcases/registration_testcase.py

```diff
@@ -43,6 +43,24 @@ def test_10_3_4_1_1_1(self):
     response = self._sas.Registration(request)['registrationResponse'][0]
     # Check registration response
     self.assertEqual(response['response']['responseCode'], 0)
+    
+  @winnforum_testcase
+  def test_10_3_4_1_2_1(self):
```

Comment:
Shall we just switch to using the new names now? I think this one should be WINFF.FT.S.REG.6.